en/debug/sensor_uorb_topic_debugging.md

```diff
@@ -0,0 +1,17 @@
+# Sensor/Topic Debugging using the Listener Command
+
+The uORB is an asynchronous `publish()` / `subscribe()` messaging API used for
+inter-thread/inter-process communication. The `listener` command can be used from the *QGroundControl MAVLink Console* to inspect topic (message) values, including the current values published by sensors.
```

Comment:
Makes sense. I think I'd still move the note up.
The QGC shell was very recently added. It's basically the same as the mavlink shell, it just can be a bit more comfortable if you want to use QGC at the same time. We can discuss this offline if you'd like.